tests/phpunit/Application/LocalizedTextLookupTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace ProfessionalWiki\WikibaseFacetedSearch\Tests\Application;
+
+use PHPUnit\Framework\TestCase;
+use MediaWiki\MediaWikiServices;
+use ProfessionalWiki\WikibaseFacetedSearch\Application\LocalizedTextLookup;
+use ProfessionalWiki\WikibaseFacetedSearch\WikibaseFacetedSearchExtension;
+use Wikibase\DataModel\Entity\Item;
+use Wikibase\DataModel\Entity\ItemId;
+use Wikibase\Lib\Tests\MockRepository;
+
+/**
+ * @covers \ProfessionalWiki\WikibaseFacetedSearch\Application\LocalizedTextLookup
+ */
+class LocalizedTextLookupTest extends TestCase {
+
+	protected function setUp(): void {
+		$this->repo = $this->newMockRepository();
+	}
+
+	public function testGetLabelFromEntityIdString() {
+		$lookup = $this->newLocalizedTextLookup();
+
+		$this->setLabelToEntity( 'Test label', 'Q1' );
+		// TODO: Might have to mock labelLookup or figure out a way to use Wikibase
```

Comment:
@JeroenDeDauw I am not sure what approach to take when it comes to testing getting labels from Wikibase entities. In my pursuit for answer in the Wikibase documentations, every pages have your name on it so I figure that you might know what would be a good approach 😅

![image](https://github.com/user-attachments/assets/a678b8c0-7573-40a0-965b-14809d203145)

Reply:
There is no need to create entities or persist anything

Reply:
It does, thanks for sharing the issue!